src/Consts/ContestDetailsCONSTS.js

```diff
@@ -0,0 +1,19 @@
+export const toBeAnnounced = 'TBA';
+
```

Comment:
maybe "wkrótce" not TBA? i think polish user won't understand what means TBA

Reply:
> ![image](https://user-images.githubusercontent.com/92270179/151585573-b8925f48-7a19-4d81-8fa2-5fe443551b41.png) it works almost ^ it is like that on ypur computer? please merge latest development to your branch but impressive!

here for test I render two Components. First is with good_data and second i with no_data (this TBA things).